packages/docs/src/repl/repl.tsx

```diff
@@ -77,27 +76,19 @@ export const Repl = component$((props: ReplProps) => {
     }
   });
 
```

Comment:
Missing `  // eslint-disable-next-line qwik/no-use-visible-task` :)

Reply:
The eslint rules don't work in the Qwik codebase because eslint can't load local plugins :(. I tried with workspace:* but got in trouble somehow.